src/TransCreator/utilityMethod.js

```diff
@@ -0,0 +1,23 @@
+import { setNewTransactionAmount } from '../Stats/actionCreator';
+
+export default function numHandler(val, dispatch) {
+  if (val === '') {
+    dispatch(setNewTransactionAmount(null));
```

Comment:
the numHandler method should be independent from redux and any framework, library. It should be a pure js method. So pls remove any reference to dispatch, you just need to return the value

Reply:
fixed